<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>469</width>
    <height>297</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>642</width>
    <height>413</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Текстовый редактор</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#IOfield {
	background-color: rgba(48, 200, 255, 0.3);
}

#workingOnLabel {
	background-color: rgba(28, 180, 235, 0.3);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="fileNameInput">
        <property name="minimumSize">
         <size>
          <width>123</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Имя файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="minimumSize">
         <size>
          <width>123</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Создать новый</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="minimumSize">
         <size>
          <width>123</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сохранить файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="minimumSize">
         <size>
          <width>123</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Открыть файл</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="workingOnLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="IOfield">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
